awx/ui/client/lib/models/Base.js

```diff
@@ -349,6 +366,32 @@ function graft (id) {
     return new this.Constructor('get', item, true);
 }
 
+function getDependentResourceCounts (id) {
+    if (this.setDependentResources) {
+        this.setDependentResources(id);
+    } else {
+        return Promise.resolve([]);
+    }
+
+    const dependentResourcePromises = [];
+
+    this.dependentResources.forEach(dependentResource => {
+        const config = {};
+
+        if (dependentResource.params) {
+            config.params = dependentResource.params;
+        }
+
+        dependentResourcePromises.push(dependentResource.model.http.get(undefined, config)
```

Comment:
`http.get(undefined, config)` -- I don't like having to pass in `undefined` as the first argument.  Maybe we can handle this in the `http.get` method itself?